i18n/pt-BR.json

```diff
@@ -0,0 +1,105 @@
+  "POWERCORD_UNLINK": "Desligar",
+  "POWERCORD_LINK_NOW": "Ligar agora",
+  "POWERCORD_LINKING_UNLINKED": "Você ainda não ligou a sua conta.",
+  "POWERCORD_LINKING_WAITING": "Ligando a sua conta...",
+  "POWERCORD_LINKING_TIMED_OUT": "Tempo do fluxo de ligação excedido :( Talvez pode tentar novamente?",
+  "POWERCORD_LINKING_ERRORED": "Um erro ocorreu. Verifique a consola para obter mais detalhes e, por favor, [reporte]({newIssueUrl}) se ocorrer novamente.",
+  "POWERCORD_COMMAND_PREFIX": "Prefixo de Comandos",
+  "POWERCORD_SETTINGS_SYNC": "Sincronizar Definições",
+  "POWERCORD_SETTINGS_SYNC_DESC": "Sincronize todas as suas definições de Powercord em múltiplos dispositivos. É preciso de uma conta Powercord!",
+  "POWERCORD_SETTINGS_NO_CLYDE": "Erradicar Clyde",
+  "POWERCORD_SETTINGS_NO_CLYDE_DESC": "Substitui [Clyde]({discordiaUrl}) nos comandos de Powercord por uma variedade de avatares e nomes de usuários selecionados pelos desenvolvedores de plugins - \"Powercord\" por padrão.",
+  "POWERCORD_SETTINGS_ADVANCED_DESC": "Não toque em coisas aqui se você não sabe o que está a fazer. Coisas inesperadas podem acontecer ao seu gato.",
+  "POWERCORD_SETTINGS_OVERLAY": "Overlay DevTools",
+  "POWERCORD_SETTINGS_OVERLAY_DESC": "@todo",
+  "POWERCORD_SETTINGS_KEEP_TOKEN": "Manter token armazenado",
+  "POWERCORD_SETTINGS_KEEP_TOKEN_DESC": "Impede que o Discord remova o seu token do localStorage, reduzindo o número de logouts indesejados.",
+  "POWERCORD_SETTINGS_TRANSPARENT": "Janela Transparente",
+  "POWERCORD_SETTINGS_TRANSPARENT_DESC": "Torna transparentes as janelas abertas pelo Discord, úteis para temas.\n****AVISO:**** Isto quebrará **window snapping** no Windows. **A acelaração do hardware** deve estar **desligada** no Linux. Você poderá encontrar problemas e ter um fundo preto em alguns casos, como quando a janela é cortada na parte superior ou inferior devido à resolução do seu monitor ou quando as devtools estão abertas e encaixadas. **Requer reinicialização**.",
+  "POWERCORD_SETTINGS_EXP_WEB_PLATFORM": "Recursos Experimentais da Plataforma Web",
+  "POWERCORD_SETTINGS_EXP_WEB_PLATFORM_DESC": "Ativa os recursos experimentais da Plataforma Web do Chromium que estão em desenvolvimento, como CSS \"backdrop-filter\". Como estes recursos estão em desenvolvimento, você poderá encontrar problemas e APIs poderão mudar a qualquer momento. **Requer reinicialização**.",
+  "POWERCORD_SETTINGS_DISCORD_EXPERIMENTS": "Ativar Experimentos no Discord",
+  "POWERCORD_SETTINGS_DISCORD_EXPERIMENTS_DESC": "****AVISO:**** A ativação desta opção fornece acesso a recursos que podem ser detetados pelo Discord e podem resultar num **encerramento da sua conta**. Powercord **não é responsável** pelo o que você faz com este recurso. Deixe-o desativado se não tiver certeza. A equipe do Powercord **não** oferecerá suporte para qualquer experimento.",
```

Comment:
Since I believe `encerramento da sua conta` is `account termination` it should be wrapped in 4 * to make it be red in the UI.
```suggestion
  "POWERCORD_SETTINGS_DISCORD_EXPERIMENTS_DESC": "****AVISO:**** A ativação desta opção fornece acesso a recursos que podem ser detetados pelo Discord e podem resultar num ****encerramento da sua conta****. Powercord **não é responsável** pelo o que você faz com este recurso. Deixe-o desativado se não tiver certeza. A equipe do Powercord **não** oferecerá suporte para qualquer experimento.",
```